hack/build/config.sh

```diff
@@ -28,8 +28,7 @@ FUNC_TEST_BAD_WEBSERVER="cdi-func-test-bad-webserver"
 FUNC_TEST_PROXY="cdi-func-test-proxy"
 # update this whenever builder Dockerfile is updated
 BUILDER_TAG=${BUILDER_TAG:-0.0.13}
-BUILDER_IMAGE=${BUILDER_IMAGE:-quay.io/kubevirt/kubevirt-cdi-bazel-builder@sha256:45ff791390bcd3902781e0f790083f30bba8553d2568e34e75675228c5d4bfb5
-}
+BUILDER_IMAGE=${BUILDER_IMAGE:-quay.io/kubevirt/kubevirt-cdi-bazel-builder:0.0.13}
```

Comment:
I think you misunderstood what I was saying, we will need to set the builder image based on the arch, something like
```bash
if [ $ARCH == "amd64" ]; then
  BUILDER_IMAGE=${BUILDER_IMAGE:-quay.io/kubevirt/kubevirt-cdi-bazel-builder@sha256:45ff791390bcd3902781e0f790083f30bba8553d2568e34e75675228c5d4bfb5
else if [ $ARCH == "arm64" ]; then
  BUILDER_IMAGE=${BUILDER_IMAGE:-quay.io/kubevirt/kubevirt-cdi-bazel-builder-arm@sha256:<sha>
fi
```
The point I was trying to make we don't know what the sha is, until after we have created the builder image. So that step has to happen after this PR is merged, and a new builder image has been pushed.

Reply:
Ok, I get your idea, but I have one unclear on this. 
As the multi-arch manifest for the builder image has been uploaded, by using a tag to identify the image, docker would automatically detect the server arch and decides which image should be used. Why use sha256 to identify the image? Are there any specific purposes?